docs/user/experiments-api.md

```diff
@@ -49,6 +47,44 @@ assertEquals(
 
 </div>
 
+<div data-lang="Python" class="tab">
+
+```Python
+from glean import Glean
+
+# Annotate Glean pings with experiments data.
+Glean.set_experiment_active(
+  experiment_id = "blue-button-effective",
+  branch = "branch-with-blue-button",
+  extra: {
+    "buttonLabel": "test"
+  }
+)
```

Comment:
The `extra` kwarg would produce a `SyntaxError` because of the `:` char:

```python
Glean.set_experiment_active(
    experiment_id="blue-button-effective",
    branch="branch-with-blue-button",
    extra={"buttonLabel": "test"},
)
```